### PR TITLE
LG-4338: Normalizes Modal size property

### PR DESCRIPTION
**Pull request by bruugey**
## ✍️ Proposed changes

[LG-4338](https://jira.mongodb.org/browse/LG-4338)

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/modal/src/Modal/ModalView.tsx [hunk 1]
```diff
@@ -87,6 +87,20 @@ const ModalView = React.forwardRef(
           fallbackFocus: `#${closeId}`, // tests fail without a fallback. (https://github.com/focus-trap/focus-trap-react/issues/91)
         };
 
+    let size: ModalSize = ModalSize.Default;
```
Comment (shaneeza) [inline packages/modal/src/Modal/ModalView.tsx:90]: Could you do something like, 
```
size = ModalSize[size] ?? ModalSize.Default
```

Comment (shaneeza) [reply packages/modal/src/Modal/ModalView.tsx:90]: nvm, i don't think thats correct

Comment (shaneeza) [reply packages/modal/src/Modal/ModalView.tsx:90]: or maybe something like this so you don't have to list every size

```
const exists = Object.values(ModalSize).includes(size);
const size = exists ? size : ModalSize.Default
```

Comment (bruugey) [reply packages/modal/src/Modal/ModalView.tsx:90]: good call - i can do this

**Review by stephl3 — APPROVED** [review]
LGTM

**Pull request merged**